lib/util/urltils.js

```diff
@@ -162,6 +162,31 @@ module.exports = {
     }
   },
 
+  /**
+   * Obfuscates path parameters with regex from config
+   *
+   * @param {Config} config The configuration containing the regex
+   * @param {string} path The path to be obfuscated
+   * @returns {string} The obfuscated path or the original path
+   */
+  obfuscatePath: function obfuscatePath(config, path) {
+    // TODO: handle SyntaxError: Invalid regular expression
```

Comment:
can you address this please?

Reply:
I'm wondering whether this is the best place to address it.
I'm reckoning we want to `logger.error` if it happens, but I can't see any other `urltils` function using the `logger` so I'm wondering whether the logic should be handler somewhere.
Seems like the most reasonable place to handle it, otherwise I would be polluting other files with unrelated logic.

Reply:
I've added functionality but I can't figure out how to stub the `logger` so there is a failing test.